src/main/webapp/style.css

```diff
@@ -8,14 +8,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Can we have a common css file across the project?
Can you please work with @kirti-d and get back.